### PR TITLE
Add Prow Jobs for cluster-network-addons-operator

### PR DESCRIPTION
**Pull request by danielBelenky**
Signed-off-by: Daniel Belenky <dbelenky@redhat.com>

### CODE REVIEW

File: github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+filters:
+  ".*":
+    reviewers:
+      - phoracek
```
Comment (phoracek) [inline github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS]: Please add `qinqon` here and to approvers.

BTW isn't this dangerous? Let's say I change my nick on GitHub and someone else takes over. Do they get approver rights everywhere?

File: github/ci/prow/files/plugins.yaml [hunk 2]
```diff
@@ -112,6 +112,13 @@ plugins:
   - lgtm
   - approve
 
+  kubevirt/cluster-network-addons-operator:
+  - trigger
+  - owners-label
+  - release-note
+  - lgtm
+  - approve
```
Comment (phoracek) [inline github/ci/prow/files/plugins.yaml]: Could you add `tide`?

Comment (rmohr) [reply github/ci/prow/files/plugins.yaml]: tide should be enabled by the change in config.yaml above.

File: github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+presubmits:
+  kubevirt/cluster-network-addons-operator:
+    - name: pull-e2e-lifecycle-k8s
```
Comment (rmohr) [inline github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml]: Please name them something like `pull-e2e-cluster-network-addons-operator-lifecycle-k8s`

File: github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.e2e-lifecycle-k8s.sh"
+    - name: pull-e2e-lifecycle-okd
```
Comment (rmohr) [inline github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml]: Please name them something like `pull-e2e-cluster-network-addons-operator-lifecycle-okd`

File: github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.e2e-lifecycle-okd.sh"
+    - name: pull-e2e-workflow-k8s
```
Comment (rmohr) [inline github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml]: rename

File: github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+              privileged: true
+            resources:
+              requests:
+                memory: "29Gi"
+            command:
+              - "/usr/local/bin/runner.sh"
+              - "/bin/sh"
+              - "-c"
+              - "automation/check-patch.e2e-workflow-k8s.sh"
+    - name: pull-e2e-workflow-okd
```
Comment (rmohr) [inline github/ci/prow/files/jobs/cluster-network-addons-operator/cluster-network-addons-operator-presubmits.yaml]: rename

File: github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS (hunk 1 above)
Comment (danielBelenky) [reply github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS]: This config gives you permission only for the `cluster-network-addons-operator/` directory. About the username change, you might be right.

Comment (rmohr) [reply github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS]: As far as I remember, you need to be at least a contributer to a repo or org

**Comment by rmohr** [comment]
/lgtm
/approve
/hold

Feel free to unhold if you are happy with the current configuration.

File: github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS (hunk 1 above)
Comment (phoracek) [reply github/ci/prow/files/jobs/cluster-network-addons-operator/OWNERS]: Thanks

**Comment by rmohr** [comment]
needs a rebase.

**Pull request merged**